docs/src/appendixes/object_stores.md

```diff
@@ -0,0 +1,424 @@
+# Appendix A - Common object stores
+
+You can archive the backup files in any service that is supported
+by the Barman Cloud infrastructure. That is:
+
+- [AWS S3](https://aws.amazon.com/s3/)
```

Comment:
Suggest making these links to the section headings below and in the sections link out to the product pages. eg

* [AWS S3](#aws_s3)

## AWS S3

Amazon's [S3](https://aws.amazon.com/s3/) object storage allows... (some general notes/guidance)....

Reply:
Done